lib/Elevate/Components/cPanelPrep.pm

```diff
@@ -0,0 +1,114 @@
+package Elevate::Components::cPanelPrep;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Components::cPanelPrep
+
+Perform tasks to ensure cPanel is in a safe state to leapp.
+
+=cut
+
+use cPstrict;
+
+use File::Slurper  ();
+use File::Basename ();
+
+use Elevate::Constants        ();
+use Elevate::StageFile        ();
+use Elevate::SystemctlService ();
+
+use Cpanel::FileUtils::TouchFile ();
+
+use Cwd           ();
+use Log::Log4perl qw(:easy);
+
+use parent qw{Elevate::Components::Base};
+
+sub pre_leapp ($self) {
```

Comment:
I wonder if we should also have a post_leapp() function that could undo some of the prep, like re-enabling the services, etc.